From 260358030731b5864ec9f8350ae902a126901479 Mon Sep 17 00:00:00 2001
From: renchenglei <chenglei.ren@intel.com>
Date: Tue, 29 Dec 2020 18:03:03 +0800
Subject: [PATCH] Disable HWC

We may encounter HWC crash when playing video with
Allocator & Mapper 4.0 on GVT-d, here we disable HWC
for a WA.

Tracked-On:
---
 common/core/gpudevice.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/core/gpudevice.cpp b/common/core/gpudevice.cpp
index 4e65d55da093..48dc86530c06 100644
--- a/common/core/gpudevice.cpp
+++ b/common/core/gpudevice.cpp
@@ -101,7 +101,7 @@ uint32_t GpuDevice::GetFD() const {
 }
 
 bool GpuDevice::IsGvtActive() const {
-  return gvt_active_;
+  return true;
 }
 
 NativeDisplay *GpuDevice::GetDisplay(uint32_t display_id) {
-- 
2.17.1

